### DISCUSSION TITLE
Child groups are not directly visible in the Groups menu

### DISCUSSION THREAD

**Discussion by mpalavrov** [keycloak/keycloak#12030] [Archive]
Hello,

In the **Groups** tab, the **Child groups** in the **New Admin Console** are not directly visible, like they are in the Old Admin Console, which might cause difficult managing when having many Child groups.
- New Admin Console: 
![image](https://user-images.githubusercontent.com/33352511/168740136-0c69a524-b80d-47fa-af6d-226fdd429772.png)
- Old Admin Console
![image](https://user-images.githubusercontent.com/33352511/168740176-6a153841-7ff4-454a-b2c4-fe27bc65b7f8.png)

What might help here is to have all Child groups listed as well in the New Admin Console, together with the main Groups, having a tag, that it is a Child group and for example a column for showing its parent group.
In case it is on purpose, and should be similar to the Composite roles in the Realm and Client Roles, then it might be good at least to have a sign, that the Group is having a Child Groups.

**Comment by edewit** [comment]
You can still search for groups it will also tell you what the group hierarchy is:
![image](https://user-images.githubusercontent.com/51133/169049635-e3ab3d05-2c6f-4aac-9109-dda2c427a201.png)
You find it under the *Action* menu in the group section

**Comment by mpalavrov** [comment]
Hello @edewit,
Thank you very much. You are right, the subgroups can be Searched there (_it was something I missed_), but in the old UI, they are listed, and then filtered, and in the new UI you need to know what to search for, so you can have listed all the groups. It might be useful to have the option to "list all". Also, it is not very intuitive, why do we have 2 Search buttons and what is the difference?
from **Groups** directly:
![image](https://user-images.githubusercontent.com/33352511/169210626-33f280e5-ff07-4e98-86fd-245f493dc650.png)
from **Groups --> Actions --> Search Group**:
![image](https://user-images.githubusercontent.com/33352511/169210679-c9df36ce-f5f1-4034-b34a-296b965cae7e.png)

**Reply by edewit** [reply]
From groups directly is more a filter that filters the current list the search on also finds subgroups that is the difference
